package.json

```diff
@@ -6,14 +6,18 @@
   "author": "Tristan Matthias <hello@tristanmatthias.com>",
   "license": "MIT",
   "private": true,
+
   "scripts": {
     "postinstall": "yarn build",
     "build": "lerna run build",
-    "start-api": "cd packages/api; yarn start"
+    "start-api": "cd packages/api; yarn start",
+    "start-api-win": "cd packages/api && yarn start"
```

Comment:
These look to be doing the same thing. Lets stick to the one for now

Reply:
This part the weird part about windows.
I can run this command straight from the CLI - "cd packages/api; yarn start"
But I cannot do "yarn start-api" or "npm run start-api"

What I can do is "cd packages/api && yarn start"

This is why I have it there.

Reply:
I see. This is mostly for the production environment anyway. For local, you should be running `yarn dev` from the `packages/api` folder